docs/DeveloperGuide.md

```diff
@@ -383,9 +379,44 @@ Command Line Interface (CLI) while still having the features of other money mana
 
 * **Mainstream OS**: Windows, Linux, Unix, OS-X
 
+## Appendix: Instructions for Manual Testing
+
+Given below are some instructions for testing the app manually. Note that they are meant to provide a starting point
+and are not an exhaustive list of test cases.
+
+### Initial Launch
+
+1. Download the jar file and copy it into an empty folder
+2. Go to that directory in the command line and run `java -jar FILENAME.jar`.
+3. The app should automatically create a data folder and the necessary files for the app to run.
+
```

Comment:
Very picky, but maybe add a line to recommend Internet connection for live exchange rates? Otherwise great description of initial launch!

Reply:
Good suggestion, I have added this